src/jbi/whiteboard_actions/default.py

```diff
@@ -84,7 +84,14 @@ def bug_create_or_update(
 
     def create_and_link_issue(self, payload):
         """create jira issue and establish link between bug and issue; rollback/delete if required"""
-        fields = {**payload.bug.map_as_jira_issue(), "key": self.jira_project_key}  # type: ignore
+        comment_list = self.bugzilla_client.get_comments(idlist=[payload.bug.id])
+        fields = {
+            **payload.bug.map_as_jira_issue(),  # type: ignore
+            "description": comment_list["bugs"][str(payload.bug.id)]["comments"][0][
```

Comment:
_added description to creation event through additional api call to get comments of bug.id_